ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -46,15 +46,23 @@ else
             "${SSHOPTS[@]}" \
             "${SHARED_DIR}/test-list" \
             "root@${IP}:/tmp/test-list"
-        echo "### Running tests"
+        echo "### Running openshift/conformance tests"
         ssh \
             "${SSHOPTS[@]}" \
             "root@${IP}" \
             openshift-tests \
             run \
             "openshift/conformance/parallel" \
             --dry-run \
-            \| grep -Ff /tmp/test-list \|openshift-tests run -o /tmp/artifacts/e2e.log --junit-dir /tmp/artifacts/junit -f -
+            \| grep -Ff /tmp/test-list \|openshift-tests run -o /tmp/artifacts/e2e_parallel.log --junit-dir /tmp/artifacts/junit -f - \
+        && ssh \
+            "${SSHOPTS[@]}" \
+            "root@${IP}" \
+            openshift-tests \
+            run \
+            "openshift/conformance/serial" \
+            --dry-run \
+            \| grep -Ff /tmp/test-list \|openshift-tests run -o /tmp/artifacts/e2e_serial.log --junit-dir /tmp/artifacts/junit -f -
```

Comment:
Is there any particular benefit in separating the runs? If not, I'd kept them together, so that also the output artifact will be just one (e2e.log and junit xml)

Reply:
Thanks for review. It was pretty useful for analysis per test suite in terms of durations, failures, etc. instead of working on one large `e2e.log` file. Especially, when the other test suites will be introduced, `e2e.log` will get larger. But if `e2e.log` usage is the convention of this repo or it is used somewhere else with this static name, I can rename it.

Reply:
Considering that the long-term goal is to cover `openshift/conformance` (and to add further tests to cover bmo/cbo) I'd prefer a single test run with a single consolidate output file